plugin.xml

```diff
@@ -1,123 +1,122 @@
-    <framework src="libsqlite3.tbd"/>
-    <framework src="libz.tbd"/>
-    <framework src="GoogleCloudMessaging" type="podspec" spec="~> 1.2.0"/>
-    <framework src="GGLInstanceID" type="podspec" spec="~> 1.2.1"/>
-  </platform>
-  <platform name="windows">
-    <hook type="after_plugin_install" src="hooks/windows/setToastCapable.js"/>
-    <js-module src="src/windows/PushPluginProxy.js" name="PushPlugin">
-      <merges target=""/>
-    </js-module>
-    <config-file target="config.xml" parent="/*">
-      <preference name="WindowsToastCapable" value="true"/>
-    </config-file>
-  </platform>
+  </description>
+
+	<license>MIT</license>
+
+	<js-module src="www/push.js" name="PushNotification">
+        <clobbers target="PushNotification" />
+	</js-module>
+
+	<engines>
+        <engine name="cordova" version=">=6.4.0"/>
+        <engine name="cordova-android" version=">=6.0.0"/>
+        <engine name="cordova-ios" version=">=4.3.0"/>
+    </engines>
+
+    <preference name="SENDER_ID" default="85075801930"/>
+
+	<!-- android -->
+	<platform name="android">
+
+		<config-file target="res/xml/config.xml" parent="/*">
+			<feature name="PushNotification" >
+				<param name="android-package" value="com.adobe.phonegap.push.PushPlugin"/>
+			</feature>
+		</config-file>
+
+        <config-file target="res/values/strings.xml" parent="/resources">
+            <string name="google_app_id">$SENDER_ID</string>
+        </config-file>
+
+        <config-file target="AndroidManifest.xml" parent="/manifest">
+            <uses-permission android:name="android.permission.INTERNET"/>
+            <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+            <uses-permission android:name="android.permission.WAKE_LOCK"/>
+            <uses-permission android:name="android.permission.VIBRATE"/>
+        </config-file>
+
+		<config-file target="AndroidManifest.xml" parent="/manifest/application">
+			<activity android:name="com.adobe.phonegap.push.PushHandlerActivity" android:exported="true"/>
+            <receiver android:name="com.adobe.phonegap.push.BackgroundActionButtonHandler"/>
+            <service
+                android:name="com.adobe.phonegap.push.FCMService">
+                <intent-filter>
+                    <action android:name="com.google.firebase.MESSAGING_EVENT" />
+                </intent-filter>
+            </service>
+            <service
+                android:name="com.adobe.phonegap.push.PushInstanceIDListenerService">
+                <intent-filter>
+                    <action android:name="com.google.firebase.INSTANCE_ID_EVENT" />
+                </intent-filter>
+            </service>
+		</config-file>
+
+        <framework src="push.gradle" custom="true" type="gradleReference" />
+        <framework src="com.android.support:support-v13:23+" />
+        <framework src="me.leolin:ShortcutBadger:1.1.11@aar"/>
+        <framework src="com.google.firebase:firebase-messaging:+" />
```

Comment:
i've done a quick scan on the plugins on ionic native and other plugins that references the google play services and this is what I found:

cordova-plugin-googleplus - ionic native
```
<framework src="com.google.android.gms:play-services-auth:+" />
<framework src="com.google.android.gms:play-services-identity:+" />      
```
cordova-plugin-googlemaps - ionic native
```
<framework src="com.google.android.gms:play-services-maps:9.8.0" />
<framework src="com.google.android.gms:play-services-location:9.8.0" />
```
cordova-plugin-google-analytics - ionic native
```
<framework src="com.google.android.gms:play-services-analytics:+" />
```

cordova-admob-pro - ionic native
```
<framework src="com.google.android.gms:play-services-ads:+" />
```

cordova-plugin-analytics
```
<framework src="com.google.android.gms:play-services-analytics:+" />
```

cordova-admob
```
<framework src="com.google.android.gms:play-services-ads:+" />
```

cordova-plugin-firebase
```<framework src="com.google.firebase:firebase-core:+" />
<framework src="com.google.firebase:firebase-messaging:+" />
<framework src="com.google.firebase:firebase-crash:+" />
<framework src="com.google.firebase:firebase-config:+" />
```

I understand even Android Studio advises again using the "+" in version numbers as it may lead to unpredicatable builds,  but then using "+" for now seems to be the only way to maintain compatibility with the other plugins out there.